lib/cc/yaml/nodes/severity.rb

```diff
@@ -0,0 +1,26 @@
+module CC
+  module Yaml
+    module Nodes
+      class Severity < Scalar
+        SEVERITIS = [
+          "info".freeze,
+          "minor".freeze,
+          "major".freeze,
+          "critical".freeze,
+          "blocker".freeze
+        ].freeze
+
+        def value=(value)
+          unless value.nil?
+            normalized_value = value.to_s.strip.downcase
+            if SEVERITIS.include? normalized_value
+              @value = normalized_value
+            else
+              error(%(Unexpected severity %s), value.inspect)
```

Comment:
I couldn't get this formatting to work locally. I think we usually pull the format string up to a constant and use `#format` elsewhere:

```ruby
error(format(INVALID_SEVERITY_ERROR, value.inspect))
```